components/log-viewer-webui/client/src/components/QueryBox/index.module.css

```
@@ -0,0 +1,27 @@
+.queryBox {
+  width: 100%;
+}
+
+.queryBoxPosition {
+  /* Enables position: absolute in Wrapper and Progress. */
+  position: sticky;
```

if i understand correctly, that we are adding `sticky` is only to create the context for any "absolute" positioned children. if so, i believe changing it as `relative` will be more proper.
```suggestion
  position: relative;
```